src/stylesheets/_layout.scss

```diff
@@ -316,41 +316,25 @@
 
 @include layout-padding($layout-spacing);
 
-.hide-xs {
-    @media (max-width: map-get($layout-breakpoints, sm) - 1) {
-        display: none !important;
-        visibility: hidden !important;
-    }
-}
-
-.hide-sm {
-    @media (max-width: map-get($layout-breakpoints, md) - 1) {
-        display: none !important;
-        visibility: hidden !important;
-    }
-}
-
-.hide-md {
-    @media (max-width: map-get($layout-breakpoints, lg) - 1) {
-        display: none !important;
-        visibility: hidden !important;
-    }
-}
+// Create the hide-{breakpoint} and show-{breakpoint} classes
+@mixin layout-hiding-rules($bp) {
+    @each $name, $value in $bp {
+        .hide-#{$name} {
+            @media (max-width: $value - 1) {
+                display: none !important;
+                visibility: hidden !important;
+            }
+        }
 
-.show-xs {
-    display: block;
+        .show-#{$name} {
+            display: block;
 
-    @media (min-width: #{map-get($layout-breakpoints, sm)}) {
-        display: none !important;
-        visibility: hidden !important;
+            @media (min-width: #{$value}) {
+                display: none !important;
+                visibility: hidden !important;
+            }
+        }
     }
 }
 
-.show-sm {
-    display: block;
-
-    @media (min-width: #{map-get($layout-breakpoints, md)}) {
-        display: none !important;
-        visibility: hidden !important;
-    }
-}
+@include layout-hiding-rules($layout-breakpoints);
```

Comment:
.hide-xs and .show-xs will now be missing as xs is not in $layout-breakpoints

Reply:
quick fix is to also add `@include layout-hiding-rules((xs: 0));`

Reply:
I've just added in the suggestion above, as I needed to rebase on this branch and this broke what I was doing

Reply:
Cool, so is the PR ok now?